doc/src/examples/algorithms/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# Boost.Geometry
+# Copyright (c) 2024 Barend Gehrels, Amsterdam, the Netherlands.
+# Use, modification and distribution is subject to the Boost Software License,
+# Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+foreach(item IN ITEMS
+    append
+    area
+    area_with_strategy
+    assign
+    assign_2d_point
+    assign_3d_point
+    assign_inverse
+    assign_points
+    buffer_with_strategies
+    centroid
+    clear
+    closest_points
+    closest_points_strategy
+    comparable_distance
+    convert
+    convex_hull
+    correct
+    densify
+    densify_strategy
+    discrete_frechet_distance
+    discrete_frechet_distance_strategy
+    discrete_hausdorff_distance
+    discrete_hausdorff_distance_strategy
+    difference
+    distance
+    envelope
+    equals
+    expand
+    for_each_point
+    for_each_point_const
+    for_each_segment_const
+    intersection_ls_ls_point
+    intersection_segment
+    intersects_linestring
+    is_simple
```

Comment:
Is this the complete list? For example, it seems that `is_empty` is missing. I am counting 82 items in the examples directory but here there are only 60 in `foreach` loop.

Reply:
Oh, I took over what was in the Jamfile.
So there are 22 more. I will add them.

Reply:
✅ added most of them. Some of them are not even used (added that as a comment).
An ancient one which was not used is deleted.